framework/templates.py

```diff
@@ -0,0 +1,11 @@
+from helpers import tf_cfg
+from string import Template
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+def fill_template(template, extra_kvs = {}):
```

Comment:
`{}` is dangerous initialisation value. It doesn't create empty object on every call, instead static object will be created and used on every call.

Reply:
Agree